apps/geo/serializers.py

```diff
@@ -148,3 +148,38 @@ class Meta:
             'region_title', 'admin_level_level', 'admin_level_title',
             'parent'
         )
+
+
+class RegionGqSerializer(ProjectPropertySerializerMixin, UserResourceSerializer):
+    project = serializers.IntegerField()
```

Comment:
Use ID, this is not a required field.

Context: project field is to attach the new region to the project. Use can also do the same from project mutations as well.